.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1 # use CircleCI 2.1
+orbs:
+   node: circleci/node@5.0.0 # Needed for javascript runtime
+   heroku: circleci/heroku@1.0.1 # Use the Heroku orb in your config
+jobs: # a collection of steps
+  build-and-test: # runs not using Workflows must have a `build` job as entry point
+    docker: # run the steps with Docker
+      - image: circleci/ruby:3.1.1 # if you're using ruby version >= 3.0, use cimg/ruby:3.x.y 
```

Comment:
It's not letting me edit, but just change circleci to cimg and keep everything after the /